modules/150-user-authn/openapi/doc-ru-config-values.yaml

```
@@ -61,6 +61,46 @@ properties:
         description:
           description: |
             Текстовое описание, содержащее информацию о том, чем этот метод аутентификации отличается от других.
+  passwordPolicy:
+    type: object
+    default: null
+    properties:
+      id:
+        description: |
+          Уникальный ID политики.
+          
+          Зарезервировано для использования в будущем.
```

```suggestion
      id:
        description: |
          Уникальный ID политики.
          
          Зарезервировано для использования в будущем.
```

I think we do not need to reserve fields for the future use, they all reserved by default.